openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -256,7 +256,7 @@ def post(self, request):
         # sending it to us. This whole bit of deserialization is kind of weird
         # though, with the renames and such. Look into this later for clennup.
         # Ref: https://github.com/openedx/edx-platform/issues/34283
-        data.pop("collection_uuid", None)
+        # data.pop("collection_uuid", None)
```

Comment:
Commenting this out doesn't break `frontend-app-library-authoring`?

If not, we should just remove the whole lines 253-259.

Reply:
It doesn't break it, I have been able to create a library, create components and edit its content.

Reply:
Updated: https://github.com/openedx/edx-platform/pull/34934/commits/837727333b14bb66602a4047bf6edea7d89863aa#diff-aee1ed7cd71a9cbd5d28d029e3589f4391e7ecc0259178a20a48cbb4f752aea5L259